solr/packaging/test/test_extraction.bats

```diff
@@ -20,6 +20,8 @@ load bats_helper
 
 setup() {
   common_clean_setup
+  # NOCOMMIT: This test must somehow start a TikaServer on random port
+  # and define a $TIKA_PORT variable
```

Comment:
Need to solve how to run this BATS test, which now needs TikaServer.

@epugh I started down a path of using `docker run` to start a tika server, mapped to SOLR_PORT+5, configure SolrCell with that address and tear it down at the end. If the system does not have docker, just pass the test with a message. Did not get it fully working but I think that may be the way to go?